collectors/python.d.plugin/mysql/README.md

```diff
@@ -260,6 +278,8 @@ remote:
 
 If no configuration is given, module will attempt to connect to mysql server via unix socket at `/var/run/mysqld/mysqld.sock` without password and with username `root`
 
+Also please note, `userstats` graph is required SUPER or PROCESS mysql priviliges.
```

Comment:
https://mariadb.com/kb/en/library/user-statistics/

i see `user statistics` plugin is disabled by default, i think we need to mention it and describe how to enable (add a link to the official docs)

Reply:
Added